### ISSUE TITLE
4 winners question 

### ISSUE THREAD

**Issue by Naylenrodriguez** [MyRainboww/Drag-Race-Simulator#83]
I noticed that now 4 queens can win in the girl group challenge. A question. Shouldn't they all win by default, not just sometimes. Like whenever they're broken off into groups, they should all receive a win every time.

**Comment by MyRainboww** [comment]
Well, in drag race herstory it has only happen 3 times (All 3 UK seasons), but in the other franchises it has never happen.

**Comment by Naylenrodriguez** [comment]
Oh so only in uk seasons it can happen. Also thanks for updating

**Comment by MyRainboww** [comment]
No, it can happen in whatever thing you simulate! But it doesn't always happen. <3

**Comment by Naylenrodriguez** [comment]
If you don't mind me asking why not every time? shouldn't, every girl group challenge have 4 winners or more. Idk it seems weird to have only one winner in that challenge. The whole point is to spread wins. But it is what it is, I hope you put it that it can have everyone in the group to win. It just makes more sense to me

**Comment by Naylenrodriguez** [comment]
Maybe you can do a vote to see, if everyone agrees. Idk maybe it's a weird Hill I'm willing to die on lol

**Comment by MyRainboww** [comment]
well it's not that eveyone agrees or not, you can go and see how many Girl Group Challenges have been done in all the seasons, and compare how many times they have give the win to the whole team and how many times just to 1 queen.

**Comment by Naylenrodriguez** [comment]
Well ty

**Comment by Naylenrodriguez** [comment]
Im regards to trigger the top 2 lipsyncs for the win. You need to do it on a computer right? You can't access it on mobile.

**Comment by MyRainboww** [comment]
yeah no problem! Yes, I think currently there are no Inspect for mobiles! :( So just for computer/laptop.

**Comment by Naylenrodriguez** [comment]
Oh dang. Well thanks so much for responding 😌 i love this simulator!!!!! Please take my multi winner suggestion to heart. Viva multi winners 💃.

Also who do you think will win all stars 7. I think trinity and monet will make it far

**Comment by MyRainboww** [comment]
I'm rooting for everyone ngl haha I'm so excietd for this new season!

**Comment by Naylenrodriguez** [comment]
Me too lol
